packages/components/src/common/storybook/customSvgIcon.tsx

```diff
@@ -1,12 +1,16 @@
 import { SvgIcon, SvgIconProps } from "@mui/material";
+import { filterProps } from "../utils";
 
 /**
  * CustomSvgIcon is a component that extends the SvgIcon component from the Material-UI library.
  * It allows easy usage of custom SVG icons with in the storybook demos.
  */
 function CustomSvgIcon(props: SvgIconProps) {
+  const ExcludeProps = ["sdsSize"];
```

Comment:
nit: since this is a constant, I think we can lift it out of the function scope like so 🙆‍♂️ ?

```ts
const EXCLUDE_PROPS = ["sdsSize"];

function CustomSvgIcon(props: SvgIconProps) {
  const finalProps = filterProps(props, EXCLUDE_PROPS);

  ...
}

```

Reply:
Ah, good catch! Thanks for pointing that out! :D